### ISSUE TITLE
Crash with fatal_handler: signum=11

### ISSUE THREAD

**Issue by dd-jacks** [FreeRDP/FreeRDP#2078]
cmake with:

```
cmake -DCMAKE_BUILD_TYPE=Release -DCHANNEL_DISP_SERVER=ON -DCHANNEL_DRIVE_SERVER=ON -DCHANNEL_PARALLEL_SERVER=ON -DCHANNEL_PRINTER=ON -DCHANNEL_RAIL_SERVER=ON -DCHANNEL_RDPEI_SERVER=ON -DCHANNEL_RDPGFX_SERVER=ON -DCHANNEL_SERIAL_SERVER=ON -DCHANNEL_SMARTCARD_SERVER=ON -DCHANNEL_TSMF_SERVER=ON -DCHANNEL_URBDRC=ON -DCHANNEL_URBDRC_CLIENT=ON -DCHANNEL_URBDRC_SERVER=ON -DWITH_ALSA=OFF -DWITH_PULSE=ON -DWITH_CUPS=ON -DWITH_DIRECTFB=ON -DWITH_PCSC=ON -DWITH_SERVER=ON .
```

seems like all dependency packages are installed.
OS: Ubuntu Gnome 14.04.1 amd64 (also tryed i386 and Debian Wheezy and have same result).

```
user@ubuntu-test:~/FreeRDP-master$ sudo xfreerdp /u:"user" /p:"password" /v:XXX.XXX.XXX.XXX /f /rfx /usb:id,dev:04b8:014a
[10:12:24:334] [7087:7088] [ERROR][com.freerdp.freerdp] - loading channel rdpdr

[10:12:24:334] [7087:7088] [ERROR][com.freerdp.freerdp] - loading channel rdpsnd

[10:12:24:334] [7087:7088] [ERROR][com.freerdp.freerdp] - loading channel drdynvc

[10:12:24:387] [7087:7088] [ERROR][com.freerdp.freerdp] - Unable to get current timezone rule

[10:12:24:564] [7087:7095] [ERROR][com.freerdp.channels.dvcman_load_addin] - Loading Dynamic Virtual Channel urbdrc

[10:12:24:572] [7087:7093] [ERROR][com.freerdp.channels.devman_load_device_service] - Loading device service drive (static)

[10:12:24:579] [7087:7095] [ERROR][com.freerdp.channels.udev_new_by_id] - VID: 0x04B8 PID: 0x014A

[10:12:24:586] [7087:7095] [DEBUG][com.freerdp.channels.udev_init] - Regist Device: Vid: 0x04B8 Pid: 0x014A InterfaceClass = 0xFF

[10:12:25:826] [7087:7095] [DEBUG][com.freerdp.freerdp] - fatal_handler: signum=11
```

It works well without device redirecting.

**Comment by akallabeth** [comment]
@dd-jacks USB redirection has some issues, I've began to refactor the code starting witn #3175, feedback welcome.

**Comment by akallabeth** [comment]
Fixed with #5745
